### DISCUSSION TITLE
Reorder issues on board

### DISCUSSION THREAD

**Discussion by yokee06** [python-gitlab/python-gitlab#1614] [Q&A]
Gitlab gives an API to move issue on the board. I couldn't find the equivalent python api.  Could anyone please clarify?

Reorder issue API: https://docs.gitlab.com/ee/api/issues.html#reorder-an-issue

**Comment by nejch** [comment]
@yokee06 I've converted this into an issue as this is not yet implemented, please follow https://github.com/python-gitlab/python-gitlab/issues/1675.
